src/java.base/share/classes/java/lang/Module.java

```diff
@@ -246,6 +250,20 @@ public ModuleLayer getLayer() {
         return null;
     }
 
+    boolean isEnableNativeAccess() {
```

Comment:
Would it be possible isEnableNativeAccess and addNEnableNativeAccess to keep them consistent with the existing methods in this file.

Reply:
Change as in "adding javadoc" ? Or is the naming of the method something that's bothering?